perception/sub8_pointcloud/include/OGridRegionType.hpp

```diff
@@ -0,0 +1,7 @@
+#pragma once
```

Comment:
I think it would be better to separate the too. In one, we are using the enum purely to draw an ogrid. In the c3, the enum is used to represent error types, that are not all necessarily tied into the ogrid. Also, I ran into a struggle of including headers from different packages.